src/server/worker/pipeline/transform/registry.go

```diff
@@ -331,18 +337,38 @@ func (reg *registry) startPipelineJob(commitInfo *pfs.CommitInfo) error {
 					ppj.logger.Logf("error incrementing restart count for job (%s): %v", ppj.pji.PipelineJob.ID, err)
 				}
 				// Reload the job's commitInfo(s) as they may have changed and clear the state of the commit(s).
-				ppj.commitInfo, err = reg.driver.PachClient().InspectCommit(ppj.commitInfo.Commit.Branch.Repo.Name, ppj.commitInfo.Commit.Branch.Name, ppj.commitInfo.Commit.ID)
+				pachClient := reg.driver.PachClient()
+				ppj.commitInfo, err = pachClient.PfsAPIClient.InspectCommit(
```

Comment:
these and others are kind of awkward since I decided to only alter functions which did logic beyond constructing the request. Just using protos everywhere would definitely be more consistent, just even more call sites to change